lib/table.js

```diff
@@ -33,6 +33,8 @@ function Table (name) {
       column.name = name
       column.nullable = options.nullable
       column.primary = options.primary
+      column.identity = options.identity
```

Comment:
Should these be covered by `objectHasProperty`?

Should we also be looking at designating these info fields as column info and handling them in some array fashion?

```js
const additionalColumnInfoFields = ['nullable', 'primary', 'identity', 'readOnly', 'length'];
additionalColumnInfoFields.forEach((f) => {
    if (objectHasProperty(options, f)) {
        column[f] = options[f];
    }
}
```

Reply:
From what I could see the `objectHasProperty` was only used for the `column.length` property because it was being set earlier in the function, only want to override that value of the `options` has something set. The rest didn't seem to exist anywhere at all.

I like your idea though, I was just trying to keep my changes to a minimum.